src/ocamlformat.ml

```diff
@@ -22,13 +22,13 @@ let moved_docstrings f c a b =
 
 (** Operations on implementation files. *)
 let impl : _ Translation_unit.t =
-  { parse= Migrate_ast.Parse.implementation
-  ; init_cmts= Cmts.init_impl
-  ; fmt= Fmt_ast.fmt_structure
-  ; equal= equal Normalize.equal_impl
-  ; moved_docstrings= moved_docstrings Normalize.moved_docstrings_impl
-  ; normalize= normalize Normalize.impl
-  ; printast= Migrate_ast.Printast.implementation }
+  { parse= Migrate_ast.Parse.use_file
+  ; init_cmts= Cmts.init_use_file
+  ; fmt= Fmt_ast.fmt_use_file
+  ; equal= equal Normalize.equal_use_file
+  ; moved_docstrings= moved_docstrings Normalize.moved_docstrings_use_file
+  ; normalize= normalize Normalize.use_file
+  ; printast= Migrate_ast.Printast.use_file }
```

Comment:
I'm tempted to change "use_file" to "toplevel_structure" or "impl" in the name of all this functions. But they would no longer match the name of the functions from OCaml's parser.

Reply:
My opinion would be that being consistent in ocamlformat itself is more important than being consistent with the parser's API, so we could have `impl` everywhere (but I don't mind `use-file` instead, as long as we are consistent)

Reply:
I renamed "use_file" to "toplevel" except in `Migrate_ast`.